packages/nimble-components/src/dialog/tests/dialog-matrix.stories.ts

```diff
@@ -60,3 +105,53 @@ export const dialogDarkThemeBlackBackground: StoryFn = createFixedThemeStory(
 );
 
 dialogDarkThemeBlackBackground.play = playFunction;
+
+export const smallDialogSizing: StoryFn = createStory(html`
+    ${createMatrix(dialogSizingTestCase, [
+        [
+            [
+                'Width Small Dialog',
+                `width: var(${dialogSmallWidth.cssCustomProperty})`
+            ]
+        ],
+        [
+            [
+                'Height Small Dialog',
+                `width: var(${dialogSmallHeight.cssCustomProperty})`
+            ]
+        ],
+        [
+            [
+                'Max Height Small Dialog',
+                `width: var(${dialogSmallMaxHeight.cssCustomProperty})`
+            ]
+        ]
+    ])}
+`);
```

Comment:
These stories aren't correct, notice they are all setting the `width:` css prop. But also it's a little odd to use the matrix helpers to render a single case.

I'd instead recommend using the createFixedThemeStory helper following a pattern like menu-button stories: https://github.com/ni/nimble/blob/main/packages/nimble-components/src/menu-button/tests/menu-button-opened-matrix.stories.ts

Reply:
Apologies for this one.. copy paste error :D

It should be okay now, I updated to use `createFixedThemeStory` as you suggested